src/ui/components/Card/index.js

```diff
@@ -18,32 +20,40 @@ export default class Card extends React.Component {
   static defaultProps = {
     // Photon is the name of the new Firefox design language. This flag
     // modifies the card style to left-align the header, add padding, and tweak
-    // the styles to be in line with the new photon mocks while we migrate
-    // the rest of the site over.
+    // the styles to be in line with the new photon mocks while we migrate the
+    // rest of the site over.
     photonStyle: false,
   }
 
   render() {
     const {
       children,
       className,
-      footerText,
+      footer: footerNode,
       footerLink,
+      footerText,
       header,
       photonStyle,
     } = this.props;
 
     let footer;
-    let footerClass;
-    if (footerText && footerLink) {
-      throw new Error(
-        'You cannot specify footerLink and footerText at the same time');
+    let footerClass = 'Card-footer';
+
+    if (
+      (footerText && footerLink) ||
+      (footerLink && footerNode) ||
+      (footerText && footerNode)
+    ) {
+      throw new Error(oneLine`You can only specify exactly one of these props:
+        footer, footerLink or footerText.`);
     } else if (footerText) {
       footer = footerText;
-      footerClass = 'Card-footer-text';
+      footerClass += ' Card-footer-text';
     } else if (footerLink) {
       footer = footerLink;
-      footerClass = 'Card-footer-link';
+      footerClass += ' Card-footer-link';
```

Comment:
We tend to always use template string for this sort of thing (even though concatenation is briefer).